packages/search/addon/components/navi-search-result/definition.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * A component that displays results for definitions
+ */
+
+import Component from '@glimmer/component';
+
+export default class NaviDefinitionSearchResultComponent extends Component {
+  /**
+   * @property {Boolean} hasMultipleDataSources
+   */
+  get hasMultipleDataSources() {
+    return new Set(this.args?.data.map(value => value.source)).size !== 1;
```

Comment:
We should check the loadedDataSources in the keg instead. Even if a query only returns results from a single data source, if there are multiple data sources it's querying, the distinction is useful.